src/content/api/notifications.mdoc

```diff
@@ -0,0 +1,45 @@
+---
+title: Notifications
+description: Notification models
+nav:
+  path: Webhooks
+  order: 1
+---
+A Notification is sent to Webhook subscribers when specific events occur in our system. Notifications may contain details about the event, and relevant metadata.
+
+## Notification Model
+
+{% properties %}
+  {% property name="id" type="string" %}
+    Notification's unique identifier.
+  {% /property %}
+
+  {% property name="type" type="string" %}
+    The [notification type](/api/notifications#notification-types).
+  {% /property %}
+
+  {% property name="data" type="string" %}
+    The associated metadata. Typically, this would be the resource state at the time of change.
```

Comment:
Suggestion: This is quite technical language. We could say something along the lines of "the details of the event" or similar to make it easier to read